### PR TITLE
Add support for the `BillingPortal` namespace and the `Session` API and resource

### PR DESCRIPTION
**Pull request by remi-stripe**
cc @stripe/api-libraries
cc @alexander-stripe

### CODE REVIEW

**Comment by remi-stripe** [comment]
I'm hitting a weird error locally 
```
$ make all
go run scripts/test_with_stripe_mock/main.go -race ./...
No custom spec file found, assuming stripe-mock is already running on port 12112
# github.com/stripe/stripe-go/v71/billingportal/session [github.com/stripe/stripe-go/v71/billingportal/session.test]
billingportal/session/client.go:17:18: undefined: stripe.BillingPortalSessionParams
billingportal/session/client.go:17:55: undefined: stripe.BillingPortalSession
billingportal/session/client.go:22:29: undefined: stripe.BillingPortalSessionParams
billingportal/session/client.go:22:66: undefined: stripe.BillingPortalSession
# github.com/stripe/stripe-go/v71/billingportal/session
billingportal/session/client.go:17:18: undefined: stripe.BillingPortalSessionParams
billingportal/session/client.go:17:55: undefined: stripe.BillingPortalSession
billingportal/session/client.go:22:29: undefined: stripe.BillingPortalSessionParams
billingportal/session/client.go:22:66: undefined: stripe.BillingPortalSession
```

I don't understand what is causing those errors at the moment and whether it's related to the change with module or the fact that this is the first namespace with 2 words.

File: billingportal/session/client.go [hunk 1]
```diff
@@ -0,0 +1,30 @@
+// Package session provides API functions related to billing portal sessions.
+package session
+
+import (
+	"net/http"
+
+	stripe "github.com/stripe/stripe-go"
```
Comment (brandur-stripe) [inline billingportal/session/client.go:7]: Try adding `v71` to the end of this like:

``` go
    stripe "github.com/stripe/stripe-go/v71"
```

Unfortunately this is needed for all stripe-go require paths now.

File: billingportal/session/client_test.go [hunk 2]
```diff
@@ -0,0 +1,18 @@
+package session
+
+import (
+	"testing"
+
+	assert "github.com/stretchr/testify/require"
+	stripe "github.com/stripe/stripe-go"
```
Comment (brandur-stripe) [inline billingportal/session/client_test.go:7]: And same with this one: `v71` at the end.

**Comment by brandur-stripe** [comment]
@remi-stripe I think it's just a minor problem with the format of your `import` paths. Comments above.

ptal @remi-stripe

**Comment by remi-stripe** [comment]
Ah gotcha, it's the bad luck of having starting the PR before the major was merged. Okay fixed all the issues, PTAL @brandur-stripe

**Pull request merged**